testing/unit/rpc/test_transaction.py

```diff
@@ -694,3 +695,172 @@
+                       '3aad96731f19ad7f6a787c0399f63822975aa94345cbdf4973af1ed97f47f3f8',
+                       '8d8cb28c58f7785621b51d220b6a1d39fe5829266495d28eaf0362dc85d7e91c'
+                       '205c1c4634604443dc566c56e1a4c0cf2eb122ac42cb482ef1436694634240c5']
+
+    expected_result = {
+        "data": [
+            {
+                "state_changes": [
+                    {
+                        "address": "000000a87cb5eafdcca6a8f82af32160bc53119b8878ad4d2117f2e3b0c44298fc1c14",
+                        "value": "ClgKKXNhd3Rvb3RoLnZhbGlkYXRvci5ibG9ja192YWxpZGF0aW9uX3J1bGVz"
+                                 "EitOb2ZYOjEsYmxvY2tfaW5mbztYYXRZOmJsb2NrX2luZm8sMDtsb2NhbDow",
+                        "type": "SET"
+                    }
+                ],
+                "events": [
+                    {
+                        "event_type": "settings/update",
+                        "attributes": [
+                            {
+                                "key": "updated",
+                                "value": "sawtooth.validator.block_validation_rules"
+                            }
+                        ]
+                    }
+                ],
+                "id": "9a2bf9a6ce2a66a3276554cf2f5b3f08a239ea4b0a80d66c3ede4f5104ea9f77"
+                      "3aad96731f19ad7f6a787c0399f63822975aa94345cbdf4973af1ed97f47f3f8",
+                "data": []
+            },
+            {
+                "state_changes": [
+                    {
+                        "address": "000000a87cb5eafdcca6a8f82af32160bc5311783bdad381ea57b4e3b0c44298fc1c14",
+                        "value": "CjAKInNhd3Rvb3RoLnZhbGlkYXRvci5iYXRjaF9pbmplY3RvcnMSCmJsb2NrX2luZm8=",
+                        "type": "SET"
+                    }
+                ],
+                "events": [
+                    {
+                        "event_type": "settings/update",
+                        "attributes": [
+                            {
+                                "key": "updated",
+                                "value": "sawtooth.validator.batch_injectors"
+                            }
+                        ]
+                    }
+                ],
+                "id": "8d8cb28c58f7785621b51d220b6a1d39fe5829266495d28eaf0362dc85d7e91c"
+                      "205c1c4634604443dc566c56e1a4c0cf2eb122ac42cb482ef1436694634240c5",
+                "data": []
+            }
+        ]
+
+    }
+
+    mock_get_list_receipts = mocker.patch('remme.shared.router.Router.list_receipts')
+    mock_get_list_receipts.return_value = return_async_value(expected_result)
+
+    request_.params = {
+        'ids': transaction_ids,
+    }
+
+    result = await list_receipts(request_)
+
+    assert expected_result == result
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('invalid_transaction_ids', (['12345', 0, 12345, True]))
+async def test_get_list_receipts_with_invalid_id(request_, invalid_transaction_ids):
```

Comment:
```suggestion
async def test_get_list_receipts_with_invalid_ids(request_, invalid_transaction_ids):
```

Reply:
@Alladin9393, this test and the one below test the same thing, instead of two tests write one.

Reply:
@anastasiia-bilova, It's two different tests, the validation check it like difference objects.